docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -14,63 +14,58 @@ endif::[]
+
+
+Note that in both cases, the server must have access to the fleet root CA for your fleet. The fleet root CA is used to sign the certificate of the connected devices.
 
-.How the server roles change depending on which method you choose:
-[caption="Figure 1: "]
-image::prov-diff-infra.png[]
+Let's take a closer look at how each method works.
 
 
-Note that in both cases, the server that plays "issuer" role, must have access to the private key and the root certificate for your fleet. The root certificate and private key are used to sign identity metadata and to verify the identity of connected devices.
+=== Provisioning with shared credentials
 
-Let's take a close look at how each method works.
+This type of provisioning is great for testing because the OTA Connect server manages the certificates and keys of the device. You can try out OTA Connect without involving the rest of your infrastructure. However, this method is not secure enough for a production scenario.
 
-== Provisioning with shared credentials
+In shared credential provisioning, the OTA Connect server creates the fleet root CA for you, and stores the private key for the CA in a Vault. You include a provisioning key for the device in the image — the “shared credential”.
 
-This method is called "provisioning with shared credentials" because you install a temporary provisioning key that is shared by all devices.
-With this method, perform the following major steps:
+In this method, the following steps occur:
 
-* You download a temporary provisioning key from the OTA Connect server and install it on a base software image.
-* You then flash your base software image to your devices.
-* Once each device boots up, it uses the shared provisioning key to request a permanent device certificate from the OTA Connect server.
-* The OTA Connect server verifies the provisioning key, issues the device with an X.509 certificate which is then downloaded to the device.
-* The device then uses this certificate for all future transactions.
+* You should download your initial provisioning key from your OTA Connect account, and bake it into the image for the device.
+* The first time the device comes online, it connects to the device gateway and says, “Hi, I’m a new device, and I don’t have a TLS certificate yet. Here’s my provisioning key.”
+* Then, our crypto service generates a new keypair and X.509 certificate for the device, signs that certificate with the private fleet root CA in the Vault instance, and sends the whole bundle to the vehicle.
+* Lastly, it deletes the vehicle’s private key.
```

Comment:
I'd argue there is another step missing here: the device needs to actually store the new keypair and cert. As a result, I might specify that the "it" in the current last step refers to the crypto service.